File: padArray.js
function padArray(array, length, filler){
  let size = length - array.length
  let arrayOne = array
  while(size > 0){
    arrayOne.push(filler)
    size --;
  }
  return arrayOne
  }
  
  padArray([1,2,3], 7, 'apple')

File: pad_array.py
def pad_array(array, length, filler):
  size = length - len(array)
  array_one = array
  while(size > 0):
    array_one.append(filler)
    size -= 1
  print(array_one)

pad_array([1,2,3], 7, 'apple')
